src/js/modules/infragistics.datasource.js

```diff
@@ -5493,6 +5497,146 @@
 
 			return fields;
 		},
+		_splitFilterExpression: function (search) {
```

Comment:
Each of the newly added methods is fully unit testable. However, I see that in the newly added unit tests, only the public method `filterByText` is invoked. Please include unit tests for all methods.